flink/README.md

```diff
@@ -35,19 +36,65 @@ No additional installation is needed on your server.
     metrics.scope.operator: flink.operator
     ```
 
-    **Note**: The system scopes must be remapped for your Flink metrics to be supported, otherwise they will be submitted as custom metrics.
+     **Note**: The system scopes must be remapped for your Flink metrics to be supported, otherwise they will be submitted as custom metrics.
```

Comment:
```suggestion
     **Note**: The system scopes must be remapped for your Flink metrics to be supported, otherwise they are submitted as custom metrics.
```